Using TensorFlow backend.
Traceback (most recent call last):
  File "prediction.py", line 406, in <module>
    main(config)
  File "prediction.py", line 387, in main
    seed=int(time.time() * 1000) % (2 ** 32 - 1))
  File "prediction.py", line 308, in data_preparing
    training_x['s92'] = [eval(x)[2] for x in training_x['s9']]
  File "prediction.py", line 308, in <listcomp>
    training_x['s92'] = [eval(x)[2] for x in training_x['s9']]
IndexError: list index out of range
Time: 2020-10-12 22:07:55.405257
[Configuration]
'est_config' : ====== Intrinsic Dim Estimation of real dataset ======
'log_dir' : exp/realcase/log
'log_filename' : est.log
'logrs' : -4:0:20

-----------------firefox----------------------
         bugID  s1  s2  s3   s4  s5  s6  s8  s90  s91  s92
0      1000002   1   1   0   32   1   0   1    0    1    0
1      1000002  13   6   7  260   2   2   3    0    1    1
2      1000002   2   1   1   10   3   3   1    0    0    1
3      1000025   5   2   3   81   1   0   2    1    0    1
4      1000025   2   1   1   27   2   3   1    0    0    1
...        ...  ..  ..  ..  ...  ..  ..  ..  ...  ...  ...
69996  1270701   9   4   5   58   2   5   2    0    1    1
69997  1270701   2   1   1   18   3   7   1    0    0    1
69998  1270701   2   1   1   20   4   7   1    0    1    0
69999  1270701   5   2   3   66   5   8   3    0    1    1
70000  1270701   2   1   1   15   6   9   1    0    0    1

[70001 rows x 11 columns]
In training set:
number of target is 11350
number of non-target is 58651
ratio of target is 0.16214054084941643
>> creating intrinsic dimension solver
>> solving...
ues L1 norm to compute the distance
candidate estiamted instrinsic dim: [ 1.69040754  1.82490198  1.87031111  2.05554985  2.15295295  2.12736527
  2.06430386  1.98596697  1.94698828  2.12543777  2.43196309  2.46894606
  2.41651349  2.42478422  2.31572103  2.17873785  1.98785259  1.74284346
  1.42796455 -0.        ]
>> task finished
-----------------chromium----------------------
